index.bs

```diff
@@ -1649,6 +1688,55 @@ partial interface MLGraphBuilder {
     **Returns:** an {{MLOperand}}. The N-D tensor of the same shape as *input* with each element casted to the target data type.
 </div>
 
+
+Casting between {{MLOperandDataType}}s is specified for some cases and [=implementation-defined=] in other cases, according to the following table:
+
+<table class=grid>
+  <caption>
+    Behavior of the {{MLGraphBuilder/cast()}} operation given the {{MLGraphBuilder/cast(input, type)/input}}'s [=MLOperand/dataType=] (rows) and target {{MLGraphBuilder/cast(input, type)/type}} (columns).
+  </caption>
+  <tr>
+    <th class=split>
+      <span class=top-right>Target type</span>
+      <span class=bottom-left>Input type</span>
+    <th>
+      {{MLOperandDataType/"float32"}},
+      {{MLOperandDataType/"float16"}}
+    <th>
+      {{MLOperandDataType/"int32"}},
+      {{MLOperandDataType/"uint32"}},
+      {{MLOperandDataType/"int64"}},
+      {{MLOperandDataType/"uint64"}},
+      {{MLOperandDataType/"int8"}},
+      {{MLOperandDataType/"uint8"}}
```

Comment:
Would explicit tag closure aid tooling (like your linting rules)? I know 'th' and 'tr' are some of those magic self-closing optional ones, but it does look weird to me :b.

Reply:
Happy to add them.

Reply:
Done in 7dc738d

(FWIW, the lint tool's parser seems to handle TR/TD/TH as mutually self-closing, it just doesn't know about DD/DT. But it doesn't hurt to add them, and makes some editors happier.)

Reply:
Appreciated.